CHANGELOG.md

```diff
@@ -418,7 +418,7 @@ make timgen-build
    - Sample SRS list for Czech Republic: `4326,3857,5514,102067,32633,32634`
    - Sample SRS list for Latvia: `4326,3857,3059`
 
-  During startup, Layman passes definitions of each EPSG to GeoServer, either from its internal sources, or from [epsg.io](https://epsg.io/). If download from epsg.io fails, warning `Not able to download EPSG definition from epsg.io` appears in log. In such case, you can [set EPSG definition manually](https://docs.geoserver.org/2.15.x/en/user/configuration/crshandling/customcrs.html) and restart GeoServer.
+  During startup, Layman passes definitions of each EPSG to GeoServer, either from its internal sources, or from [epsg.io](https://epsg.io/). If download from epsg.io fails, warning `Not able to download EPSG definition from epsg.io` appears in log. In such case, you can [set EPSG definition manually](https://docs.geoserver.org/2.20.x/en/user/configuration/crshandling/customcrs.html) and restart GeoServer.
```

Comment:
The closest to 2.15.2 is https://docs.geoserver.org/2.19.x/en/user/configuration/crshandling/customcrs.html